docs/data_node/extract_node/kafka.md

```diff
@@ -3,6 +3,157 @@ title: Kafka
 sidebar_position: 4
 ---
 
-## Configuration
-the Dashboard has not supported extracting data from Kafka for this version, 
-you can create Kafka data streams from the background via the [Command-line Tools](user_guide/command_line_tools.md).
\ No newline at end of file
+## Kafka Extract Node
+
+The `Kafka Extract Node` supports to read data from Kafka topics. It can support read data in the normal fashion and read data in the
+upsert fashion. The `upsert-kafka` connector produces a `changelog stream`, where each data record represents an `update` or 
+`delete` event. The `kafka-inlong` connector can read data and metadata.  
+
+## Supported Version
+
+| Extract Node                | Kafka version |                                                                                                                                                                                                                                                                                                                                                                                           
+|-----------------------------|---------------|
+| [Kafka](./kafka.md)         | universal     |  
```

Comment:
The kafka version  is best to be consistent with the overview.

Reply:
ok